test/test_issue1141.py

```diff
@@ -0,0 +1,27 @@
+import unittest
+from io import BytesIO
+
+from rdflib import Graph
+from rdflib.plugins.memory import IOMemory
+from rdflib.plugins.stores.auditable import AuditableStore
+
+
+class TestIssue1141(unittest.TestCase):
+    """
+    Tests is Turtle and TriG parsing works with a store with or without formula support
+    """
+    def test_issue_1141(self):
+        file = b"@prefix : <http://example.com/> . :s :p :o ."
+
+        for format in ("turtle", "trig"):
+            # with formula
+            graph = Graph()
+            self.assertTrue(graph.store.formula_aware)
+            graph.load(BytesIO(file), format=format)
```

Comment:
Just FYI, in RDFLib 5.1.0 the `graph.load()` method will be deprecated as part of the Graph API cleanup process, and in 6.0.0 `load()` will be removed.
You will need to use `graph.parse()` instead. In this case where `file` is a string with the RDF data, use `graph.parse(data=file)`.

Reply:
Ok ! Thank you! I'll fix it if the global PR design is accepted